### PR TITLE
Merge 1.7.7.x into develop

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge 177x to develop 08/03/21
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | GA tests green and Hail to all women in the world
| Possible impacts? | ~

Fusion automatique de themes/classic/templates/catalog/_partials/miniatures/product.tpl
CONFLIT (contenu) : Conflit de fusion dans themes/classic/templates/catalog/_partials/miniatures/product.tpl
Fusion automatique de themes/classic/_dev/css/components/featuredproducts.scss
CONFLIT (contenu) : Conflit de fusion dans themes/classic/_dev/css/components/featuredproducts.scss
Fusion automatique de tests/UI/pages/FO/home/index.js
Fusion automatique de tests/UI/pages/FO/FObasePage.js
CONFLIT (contenu) : Conflit de fusion dans tests/UI/pages/FO/FObasePage.js
Fusion automatique de tests/UI/pages/BO/orders/view.js
CONFLIT (contenu) : Conflit de fusion dans tests/UI/pages/BO/orders/view.js
Fusion automatique de tests/UI/pages/BO/catalog/products/add.js
Fusion automatique de controllers/admin/AdminStatusesController.php
Fusion automatique de classes/order/OrderState.php

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23564)

### CODE REVIEW

**Comment by matks** [comment]
Sanity tests failing
```
  1) Order a product and check order confirmation
       should go to login page:
     TypeError: this.clickOnHeaderLink is not a function
      at Home.goToLoginPage (pages/FO/FObasePage.js:96:16)
      at Context.<anonymous> (campaigns/sanity/06_checkoutFO/01_orderProduct.js:49:20)
```

**Review by matks — COMMENTED** [review]
Looks like sanity test are failing, there might be a conflict :/

File: tests/UI/pages/FO/FObasePage.js [hunk 1]
```diff
@@ -23,19 +24,7 @@ module.exports = class FOBasePage extends CommonPage {
...
-    this.newProductsLink = '#link-product-page-new-products-1';
-    this.bestSalesLink = '#link-product-page-best-sales-1';
-    // Our company links selectors
-    this.deliveryLink = '#link-cms-page-1-2';
-    this.legalNoticeLink = '#link-cms-page-2-2';
-    this.termsAndConditionsOfUseLink = '#link-cms-page-3-2';
-    this.aboutUsLink = '#link-cms-page-4-2';
-    this.securePaymentLink = '#link-cms-page-5-2';
-    this.contactUsLink = '#link-static-page-contact-2';
+    // footer
```
Comment (boubkerbribri) [inline tests/UI/pages/FO/FObasePage.js]: Should not be deleted, that why Sanity couldn't work.

**Review by matks — DISMISSED** [review]

**Review by matks — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @sowbiba

**Pull request merged**